backend/de.metas.manufacturing/src/main/sql/postgresql/ddl/functions/PP_Product_BOM_Recursive.sql

```diff
@@ -56,11 +49,11 @@ WITH RECURSIVE bomNode AS ((SELECT ARRAY [1::integer]                      AS pa
                            --
                            UNION ALL
                            --
-                           (SELECT parent.path || (ROW_NUMBER() OVER (PARTITION BY bomLine.PP_Product_BOM_ID, parent.path ORDER BY bomLine.line, bomLine.PP_Product_BOMLine_ID))::integer AS path,
-                                   parent.path                                                                                                                               AS parent_path,
-                                   parent.depth + 1                                                                                                                          AS depth,
-                                   bomLineProduct.Value                                                                                                                      AS ProductValue,
-                                   COALESCE(pt.Name, bomLineProduct.Name)                                                                                                    AS ProductName,
+                           (SELECT parent.path || (ROW_NUMBER() OVER (PARTITION BY bomLine.PP_Product_BOM_ID ORDER BY bomLine.PP_Product_BOMLine_ID))::integer AS path,
```

Comment:
this is probably also needed - see https://github.com/metasfresh/metasfresh/commit/224a251d9c51a6889b4f500bf508031a7e35c812

Reply:
added